source/dynamical_systems/README.md

```diff
@@ -240,25 +238,10 @@ in the constructor, the limit cycle has a constant radius.
 If an elliptical limit cycle is desired, the DS can be constructed directly from an `Ellipsoid` type.
```

Comment:
Not true anymore as this can only be achieved by setting the `limit_cycle` parameter